.github/workflows/check-csv-ordering.yaml

```diff
@@ -0,0 +1,28 @@
+name: Check CSV order
+
+on:
+
+  push:
```

Comment:
The pipeline didn't seem to run on my fork without this part. We'd want to remove it before merging.

Reply:
I got this to work by using the syntax:

```yaml
on: pull_request
```
 AND having the workflow file in the target branch of the PR.

(It also worked with a branches stanza, but I was curious what didn't work in or original so I tried various things, I think this is the simplest for this case.)

See PRs #3 through #5 in [my repo](https://github.com/johnhelmuth/fate-product-list/pulls?q=is%3Apr), and branches `test-ci-check-csv-ordering`, `test-ci-check-csv-ordering-should-fail`, and `test-ci-check-csv-ordering-should-succeed`.

Reply:
I created a PR against your repo's branch with that configuration so you can decide if that's how you want it to work... merge that if you want, it should update this PR, then we can merge this one.